specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/webapplicationfirewall.json

```diff
@@ -346,10 +350,32 @@
         }
       }
     },
+    "ManagedRulesDefinition": {
+      "description": "Allow to exclude some variable satisfy the condition for the WAF check.",
+      "required": [
+        "managedRuleSets"
+      ],
+      "properties": {
+        "exclusions": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/OwaspCrsExclusionEntry"
+          },
+          "description": "Describes the Exclusions that are applied on the policy."
+        },
+        "managedRuleSets": {
+          "type" : "array",
+          "items": {
+            "$ref": "#/definitions/ManagedRuleSet"
+          },
+          "description": "Describes the ruleSets that are associated with the policy."
+        }
+      }
+    },
     "PolicySettings": {
       "description": "Defines contents of a web application firewall global configuration.",
       "properties": {
-        "enabledState": {
+        "state": {
```

Comment:
>state [](start = 9, length = 5)

It looks like you are changing a name of a property in a stable API version, which is a breaking change. You should introduce this in a new API version.

Reply:
2019-08-01 is the new API version